ArduSub/mode.h

```diff
@@ -119,66 +119,6 @@ class Mode
     float &G_Dt;
 
 public:
-    // Navigation Yaw control
```

Comment:
FYI @Williangalvani, it looks like we had some unused code in Sub.  It compiles so it should be save to remove it.